FirebaseInAppMessaging/Swift/Source/SwiftUIPreviewHelpers.swift

```diff
@@ -73,6 +74,30 @@ public struct InAppMessagingPreviewHelpers {
     )
   }
 
+  public static func defaultBackgroundColor() -> UIColor {
```

Comment:
Hmm, could we do these inline by any chance? Otherwise these are now part of the public API surface of the product which we don't want (this is a limitation that the default value needs to have the same visibility).

If not, maybe we change it to make it colours available on all platforms.

Reply:
I couldn't figure out a good way to inline.  Any suggestions on colo(u)rs?

Reply:
I changed to colors for both, since I can't figure out a good way to make the API use a platform-specific initializer without expanding the public API.

More testing is needed to verify the colors work, but I'd like to merge this here to fix CI

@ryanwilson PTAL

Reply:
Verify deferred to fix for #8416